apps/content/content/docs/contract-first.mdx

```diff
@@ -28,20 +28,21 @@ import { oz } from '@orpc/zod'
 import { z } from 'zod'
 
 export const contract = oc.router({
```

Comment:
If you have time, can you change thing that only call `.router` to plain object? If `.router` only call standalone, and no more additional middlewares or prefix is added, this can safely use plain object instead.

Reply:
I think that's a valid point tho maybe showing that you can embed the `.router` inside others and that it's valid may be best still?